test/diagram_ref.dot

```diff
@@ -1,56 +1,56 @@
-19 [ color="#5DADE2",style="filled",penwidth="2.0",fillcolor="#dddddd",name="temp",label="Temperature",shape="record",];
-20 [ color="#5DADE2",style="filled",penwidth="2.0",fillcolor="#dddddd",name="age",label="Age",shape="record",];
-21 [ color="#5DADE2",style="filled",penwidth="2.0",fillcolor="#dddddd",name="dGDP",label="Economic Growth",shape="record",];
-22 [ color="#5DADE2",style="filled",penwidth="2.0",fillcolor="#dddddd",name="cases",label="Flu\nCases",shape="record",];
-23 [ color="#5DADE2",style="filled",penwidth="2.0",fillcolor="#dddddd",name="prices",label="Vacc\nPrice",shape="record",];
-24 [ color="#5DADE2",style="filled",penwidth="2.0",fillcolor="#dddddd",name="regres",label="Regression",shape="record",];
-25 [ color="#000000",style="filled",penwidth="2.0",fillcolor="#dddddd",name="html",label=<<U><B>Title </B></U> <BR/> Some text.>,shape="record",];
-1 -> 19 [ color=orange, dir=none, penwidth=4.0, style=solid, ]
-3 -> 15 [ color=orange, dir=none, penwidth=4.0, style=solid, ]
-3 -> 20 [ color=orange, dir=none, penwidth=4.0, style=solid, ]
-4 -> 21 [ color=missing, dir=none, penwidth=missing, style=missing, ]
-5 -> 22 [ color=orange, dir=none, penwidth=4.0, style=solid, ]
-6 -> 5 [ color=missing, dir=none, penwidth=missing, style=missing, ]
-7 -> 9 [ color=missing, dir=none, penwidth=missing, style=missing, ]
-9 -> 8 [ color=missing, dir=none, penwidth=missing, style=missing, ]
-10 -> 8 [ color=missing, dir=none, penwidth=missing, style=missing, ]
-11 -> 8 [ color=missing, dir=none, penwidth=missing, style=missing, ]
-12 -> 2 [ color=missing, dir=none, penwidth=missing, style=missing, ]
-12 -> 8 [ color=missing, dir=none, penwidth=missing, style=missing, ]
-13 -> 7 [ color=black, dir=none, penwidth=2.0, style=dashed, ]
-13 -> 18 [ color=orange, dir=none, penwidth=4.0, style=solid, ]
-14 -> 7 [ color=black, dir=none, penwidth=2.0, style=dashed, ]
-14 -> 18 [ color=orange, dir=none, penwidth=4.0, style=solid, ]
-15 -> 7 [ color=black, dir=none, penwidth=2.0, style=dashed, ]
-15 -> 18 [ color=orange, dir=none, penwidth=4.0, style=solid, ]
-16 -> 14 [ color=missing, dir=none, penwidth=missing, style=missing, ]
-17 -> 13 [ color=missing, dir=none, penwidth=missing, style=missing, ]
-18 -> 5 [ color=orange, dir=none, penwidth=4.0, style=solid, ]
-19 -> 13 [ color=orange, dir=none, penwidth=4.0, style=solid, ]
-20 -> 14 [ color=orange, dir=none, penwidth=4.0, style=solid, ]
-21 -> 23 [ color=missing, dir=none, penwidth=missing, style=missing, ]
-22 -> 24 [ color=orange, dir=none, penwidth=4.0, style=solid, ]
-23 -> 24 [ color=missing, dir=none, penwidth=missing, style=missing, ]
-24 -> 2 [ color=orange, dir=none, penwidth=4.0, style=solid, ]
-25 -> 4 [ color=black, dir=none, penwidth=2.0, style=solid, ]
+pack="true";
+1 [ color="orange", style="filled", penwidth="2.0", fillcolor="#dddddd", name="weather", label="NOAA\nWeather", shape="record", ];
```

Comment:
Why are there new spaces?

Reply:
To me, it improves readability, and I was working with the Dot source files. If you don't look at the sources and only send them directly to Graphviz, you don't even know the spaces are there...

Reply:
What I mean is, does the export suddenly add spaces everywhere?

Reply:
If so, that makes it breaking for a lot more people

Reply:
> What I mean is, does the export suddenly add spaces everywhere?

Yeah, I think so.

> If so, that makes it breaking for a lot more people

I don't want to bash anyone or be dismissive, only trying to ground your words but... 😺

This issue was the first thing I stumbled upon when I tried to use the library for pretty trivial visualization. The fact I was the first to encounter it kinda hints the library doesn't have that many users... Also, this PR has been hanging here for a year already and still hasn't generated much rage. 😉

At any rate, I don't expect many users to commit automatically generated artefacts into repositories of something...

Reply:
Fair enough, the PR is breaking anyway because it adds quotes, so we might as well go all the way.
And you're right, JuliaGraphs maintenance is rather sporadic, but we're doing our best ;)